src/routes/core/fundamentals/commands-usage/+page.md

```diff
@@ -62,61 +63,70 @@ If you're looking for the [Multiverse-Portals](/portals/fundamentals/commands-us
 
 ## Introduction
 
-In Multiverse, we have included a handy new in-game help system. To access this help, simply type `/mv`. In order to properly read this command reference, we'll walk through reading the "usage" of a command.
+There is an in-game help system! To access this, type `/mv`. This section will break down how to read commands.
```

Comment:
I'm pretty sure `/mv` won't show help in MV5 and it is `/mv help <COMMAND>` now. This would need checking in game.